pkg/idtools/idtools_unix.go

```diff
@@ -84,3 +96,125 @@ func accessible(isOwner, isGroup bool, perms os.FileMode) bool {
 	}
 	return false
 }
+
+// LookupUser uses traditional local system files lookup (from libcontainer/user) on a username,
+// followed by a call to `getent` for supporting host configured non-files passwd and group dbs
+func LookupUser(username string) (user.User, error) {
+	// first try a local system files lookup using existing capabilities
+	usr, err := user.LookupUser(username)
```

Comment:
I had a version of the code that tried not to "hide" the orig. error but it was rather ugly, and in the end, ends up as a "can't find user" (getent would have done the same searching of local filesystem in default mode, so same error). If we think there are other potential edge case errors in this path that need to be exposed we can re-work